test/e2e/specs/demo-content-with-keyboard.test.js

```diff
@@ -0,0 +1,345 @@
+}
+
+async function navigateToNextRegion() {
+	await page.keyboard.down( 'Alt' );
+	await page.keyboard.down( 'Shift' );
+	await page.keyboard.type( 'n' );
+	await page.keyboard.up( 'Shift' );
+	await page.keyboard.up( 'Alt' );
+}
+
+async function createBlockAfter() {
+	await page.keyboard.down( META_KEY );
+	await page.keyboard.down( 'Alt' );
+	await page.keyboard.type( 'y' );
+	await page.keyboard.up( 'Alt' );
+	await page.keyboard.up( META_KEY );
+}
+
+jest.setTimeout( 1000000 );
+
+describe( 'Demo content post', () => {
+	beforeAll( async () => {
+		await newPost();
+	} );
+
+	it( 'should be created with keyboard', async () => {
+		// Title of the page
+		await page.keyboard.type( 'Welcome to the Gutenberg Editor' );
+
+		// Creating the cover image
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.type( '/cover' );
+		await page.keyboard.press( 'Enter' );
+
+		// Apply center alignment (wide alignment is not available in the default theme)
+		await pressWithModifier( 'Alt', 'F10' );
+		await pressTimes( 'Tab', 2 );
+		await page.keyboard.press( 'Enter' );
+
+		// Upload the image
+		await pressTimes( 'Tab', 4 ); // We should have a way to focus the block content
+		await page.keyboard.press( 'Enter' );
+		await uploadImageInTheMediaLibrary( 'cover-1.jpg' );
+		await page.click( '.media-modal button.media-button-select' ); // validating the media gallery is not easy with keyboard
+
+		// Enter the text of image
+		await page.keyboard.press( 'Tab' );
+		await page.keyboard.type( 'Of Mountains & Printing Presses' );
+
+		// Create the next paragraph
+		await page.keyboard.press( 'Tab' );
+		await page.keyboard.type( 'The goal of this new editor is to make adding rich content to WordPress simple and enjoyable. This whole post is composed of ' );
+		await pressWithModifier( META_KEY, 'i' );
+		await page.keyboard.type( 'pieces of content' );
+		await pressWithModifier( META_KEY, 'i' );
+		await page.keyboard.type( '—somewhat similar to LEGO bricks—that you can move around and interact with. Move your cursor around and you’ll notice the different blocks light up with outlines and arrows. Press the arrows to reposition blocks quickly, without fearing about losing things in the process of copying and pasting.' );
+
+		// Create the second paragraph
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.type( 'What you are reading now is a ' );
+		await pressWithModifier( META_KEY, 'b' );
+		await page.keyboard.type( 'text block' );
+		await pressWithModifier( META_KEY, 'b' );
+		await page.keyboard.type( ' the most basic block of all. The text block has its own controls to be moved freely around the post..' );
+
+		// Create the aligned right paragraph
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.type( '... like this one, which is right aligned.' );
+		await page.mouse.move( 200, 300, { steps: 10 } ); // This shouldn't be necessary to show the toolbar
```

Comment:
🤔

Reply:
yes, mentioned in the description. We need to fix that.

Reply:
If you select the text you can get around it for now as it will show the toolbar.

Reply:
Even just command + A for now.

Reply:
My hot take: using (and thus highlighting) awkward interactions required by keyboard-only usage is better than using the mouse. 🤷‍♂️

Reply:
All the blocks don't have "text" so "command + A" won't always work. (Embed block for instance).